src/app/restaurant/service/restaurant.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RestaurantService {
+  apiUrl = 'http://localhost:3000/api';
+
+  constructor(public http: HttpClient) {
+    console.log('Hello RestServiceProvider Provider');
```

Comment:
Please remove `console.log`, if you want it badly, create wrapper for so that you can e.g `this.logger.debug("helo")`